css/style.css

```diff
@@ -51,105 +64,243 @@ ul {
-    padding: 0 11px 0 0;
     content: url(.//img/cart.svg);
+    position: absolute;
+    left: 0px;  
 }
-.user-buttons {
+.user-purchase {
     display: flex;
     justify-content: space-between;
+    width: 220px;
+    padding: 0;
 }
-.user-buttons ul {
+.nav {
     display: flex;
-    justify-content: center;
-    padding: 0 20px;
+    align-items: center;
+    justify-content: space-between;
+    width: auto;
+    padding-bottom: 58px;
+    margin-left: 60px;
+    margin-right: 60px;
+    font-size: 18px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
 }
-.user-purchase li {
-    padding: 0 0 0 17px;
+.div-catalog {
+    display: flex;
+    justify-content: left;
+    width: 500px;
+    
 }
-.nav-list {
+.div-services {
     display: flex;
+    justify-content: space-between;
+    width: 500px;
 }
-.nav-list li {
+.ul-catalog {
+    padding: 0;
+}
+.ul-services {
     display: flex;
-    align-items: center;
-    padding: 0 20px;
-    font-size: 18px;
-    text-transform: uppercase;    
+    justify-content: space-between;
+    width: 270px;
+    padding: 0;
 }
-.nav-catalog {
+.li-catalog {
     position: relative;
 }
-.nav-catalog::after {
+.li-catalog::after {
     content: url(.//img/plus.svg);
     position: absolute;
-    right: -13px; 
+    right: -32px; 
     bottom: -2px;
     opacity: 30%;    
 }
 .dropdown-menu {
     display: flex;
-    width: 1160px;
+    /* width: 1160px;
     height: 129px;
-    margin: 0 140px 0 140px;    
+    margin: 0 140px 0 140px;     */
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
     line-height: 36px;
-    background-color: #f7e184;
+    background-color: #f7e296;
+}
+.section-slider {
+    max-width: 1160px;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+}
+.section-slider {
+    position: relative;
+    margin-bottom: 115px;
 }
 .slider-item {
     display: flex;
     align-items: center;
-    justify-content: space-evenly;
+    justify-content: space-between;
+    margin-bottom: 115px;
+}
+.slider-item:last-child {
+    margin-bottom: 0;
+}
+.section-slider::before {
+    content: "";
+    display:block;
+    position: absolute;
+    height: 90px;
+    width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #f7e296;
+    z-index: -1;
+}
+.slider-img {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 560px;
+}
+.slider-content {
+    width: 560px;
+}
+.slider-content-number {
+    position: absolute;
+    margin: 0;
+    top: 40px;
+    right: 40px;
+    font-size: 179px;
+    color: #ffffff;
 }
 p.content-tagline {
     font-size: 47px;
     line-height: 56px;
 }
 p.content-description {
+    margin-bottom: 45px;
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
     line-height: 30px;
 }
+.content-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 220px;
+    height: 40px;
```

Comment:
don't use fixed height for such elements. your height shoul be defined only by font-size, line-height, and paddings